README.md

```diff
@@ -21,10 +21,11 @@ A non-exhaustive list of projects to be known to have rather unpleasant vibes.
   & [Mirai](https://github.com/etil2jz/Mirai), "The combined power of Paper forks for maximum instability!" - Tossed
   salad of people that haven't even really understood the patch system. And
   yes, *[it's that bad](https://github.com/kennytv/Yaptapia)*
+* [CatServer](https://github.com/Luohuayu/CatServer)/[FoxServer](https://github.com/Luohuayu/FoxServer), "Steals code from other hybirds and breaks their license" - Gets angry when mohist and magma DMCA them for [breaking](https://github.com/Luohuayu/FoxServer/issues/7#issuecomment-1100009357) their license
 * [Mohist](https://github.com/MohistMC/Mohist), "Get the worst parts of Bukkit and Forge and mash them together"
-  - [Swapped Essentials' plugin jar](https://github.com/MohistMC/Mohist/blob/70a303f4d02e9480cc5472c1c26f7d9cb6560732/src/fmllauncher/java/com/mohistmc/AutoDeletePlugins.java#L20-L22)
+  - (in the past) [Swapped Essentials' plugin jar](https://github.com/MohistMC/Mohist/blob/70a303f4d02e9480cc5472c1c26f7d9cb6560732/src/fmllauncher/java/com/mohistmc/AutoDeletePlugins.java#L20-L22)
```

Comment:
Not really needed. "Swapped" already has a past connatation (was it connatation?) so this addition is somewhat pointless...

Reply:
oops